marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/entitySearch/saveSavedQuery.sjs

```diff
@@ -24,11 +24,9 @@ var saveQuery;
 var userCollections = ["http://marklogic.com/data-hub/saved-query"];
 var queryDocument = JSON.parse(saveQuery);
 
-let authorized = new Security().getRolesAndAuthorities().authorities.includes("canManageSavedQuery");
-
-if(!authorized) {
-    ds.throwForbidden(xdmp.getCurrentUser() + " user doesn't have authority to save or update query");
-}
+const sec = new Security();
```

Comment:
I agree with not using "Security" as the variable name here, but I think we need something besides "sec" because a reader could easily think that's an ML function that starts with "sec.". How about "dhsec", or "hubsec"? (hubsec is kind of cool, like infosec).

Reply:
good point. will address that.